config/node4.toml

```diff
@@ -0,0 +1,39 @@
+[parity]
+chain = "pos-contracts/spec.json"
+base_path = "parity-data/node4"
+
+[ui]
+disable = true
+
+[network]
+port = 30304
+
+[rpc]
+apis = ["web3", "eth", "pubsub", "net", "parity", "parity_set", "parity_pubsub", "traces", "rpc", "shh", "shh_pubsub"]
+port = 8544
```

Comment:
This causes nodes 4 through 6 to fail to start, since the socket is already in use.

We should really be using IPC rather than a port anyway.

Reply:
They started OK during my tests. If the socket as configured in the TOML file is already in use on your computer, you could change the socket parameter to some unused value. I agree that constant port numbers are not the best solution for testing. However unless we generate TOML files to fit the unassigned ports, we have to assign constant ports, only choose numbers that are free in your system.

Reply:
@vkomenda Nevermind.  They started okay for me too, once I killed all running Parity instances.